packages/expo-modules-autolinking/scripts/ios/packages_config.rb

```diff
@@ -0,0 +1,15 @@
+require 'singleton'
+
+module Expo
+  # This class is used to store the configuration of the packages that are being used in the project.
+  # It is a singleton class, so it can be accessed from anywhere in the project.
+  class PackagesConfig
+    include Singleton
```

Comment:
maybe we don't necessarily to use singleton but class attribute since it's only a single attribute

like the easy solution though 👏

Reply:
I’ve been considering it, and I believe we should go with a singleton for two reasons:
1. It is thread-safe. Since I'm not entirely sure how CocoaPods operates internally, this seems like a safer choice.
2. We plan to add more properties and functions when we configure additional packages.